docs/routing/route_configuration_options.md

```diff
@@ -2,86 +2,108 @@
 
 ## Route domains
 
-ILC will check the exact match of domain names, accordingly, `secure.example.com` is not equal to `example.com`, so you may need to add the necessary subdomains.
+ILC checks the exact match of domain names. It means that `secure.example.com` is not equal to `example.com`, and you may need to add the necessary subdomains to handle this case properly.
 
-ILC can handle requests coming from multiple domains, so that you don't need to roll out individual instances of the ILC for every domain, you just can use one instance for all of them.
+ILC can handle requests coming from multiple domains, so you can use a single ILC instance to handle them instead of rolling out individual instances for every domain.
 
 ![ILC registry domains field](../assets/routes/domain-field.png)
 
-- Domain name, must be without protocol, etc.
+!!! note ""
+    - Domain name must be without protocol.
+    - Empty `Domain Name` field means the main app domain.
+    - To add a new domain, go to the **Router domains** section in the sidebar.
+        
+        ??? tip "Add a new domain"
+            ![ILC registry domains menu](../assets/routes/domain-create.png)
 
-- You can add new domain in **Router domains** menu.
+!!! note ""
+    The application renders only one domain at the same time. To add one header to several domains, you need to create the same route several times specifying the required domain for each route.
 
-![ILC registry domains menu](../assets/routes/domain-create.png)
+    For example:
 
-- Empty `Domain Name` field is equal to main app domain.
+    ![ILC registry domains example](../assets/routes/domain-example.png)
 
-- The application renders only one domain at the same time, so in order to add one header to several different domains, you need to create a route several times specifying the required domain for each.
+    where:
 
-**For example:**
+    * :fontawesome-solid-square:{ .color-green } - render for the main domain
+    * :fontawesome-solid-square:{ .color-red } - render for the `127.0.0.1` domain only
 
-![ILC registry domains example](../assets/routes/domain-example.png)
-![color box](../assets/routes/green-box.png) - render for main domain.
-![color box](../assets/routes/red-box.png) - render only for `127.0.0.1` domain.
-
-- You can find more information about domain [here](../multi-domains.md)
+More information about domains is available in the [Multi-domains page](../multi-domains.md).
 
 ## Route template
 
-**Important!** There must be at least one template in the routing chain.
-
-The template is needed to build the structure of our page. So if you don't create a template, ILC won't know exactly where to render the content and will show you an error instead of the desired page.
+Template is an HTML file that is used to build the structure of your page. Template is **mandatory**. If it is missing, ILC won't be able to render your content properly and will throw an error.
```

Comment:
Template is not a mandatory option

Reply:
What about the very first template? It says, that ILC won't be able to properly render the content. I believe that template is something that you need at least once.

Reply:
Clarified with @DammneD